VocabularyTrainer/Home Screen/CollectionViewCell.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  CollectionViewCell.swift
+//  VocabularyTrainer
+//
+//  Created by skrr on 29.12.22.
+//  Copyright © 2022 mic. All rights reserved.
+//
+
+import UIKit
+
+class CollectionViewCell: UICollectionViewCell {
+
+    typealias Dimensions = LanguageCellViewModel.Dimensions
+    typealias Colors = LanguageCellViewModel.Colors
+
+    private let titleLabel: UILabel = {
```

Comment:
Maybe you could use the createLabel method that is an extension of UILabel.
It would be:

```
private let titleLabel: UILabel = .createLabel(font: .preferredFont(forTextStyle: .headline),
                                               accessibilityTrait: .header,
                                               fontColor: HomeViewModel.Colors.title)
```

Reply:
oh right, forgot about that! thanks, will change that :)

Reply:
updated `UILabel+Extensions` to support init with color names and `UIColors` and refactored `CollectionViewCell` in: 54d50af1aa2377f5023e3721449aa1fd1c1b9645